test/parallel/test-fs-watch-recursive.js

```diff
@@ -33,14 +63,14 @@ tmpdir.refresh();
 
   const watcher = fs.watch(testDirectory, { recursive: true });
   let watcherClosed = false;
-  watcher.on('change', function(event, filename) {
+  watcher.on('change', common.mustCallAtLeast(function(event, filename) {
```

Comment:
I don't think we need this change in here. The test will fail if this function is not called, because the exit condition is in the callback. Can you remove all changes related to this?